messages/extend-config-missing.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the config "<%- configName %>" to extend from. Please, check the name of the config is correct.
```

Comment:
Nitpick about the wording here: `Please, check the name of the config is correct.` -> `Please check that the name of the config is correct.`